apps/sms/index.html

```diff
@@ -148,6 +155,33 @@ <h1 data-l10n-id="messages"></h1>
           <a id="threads-settings-button" data-icon="settings" data-l10n-id="settingsButton"></a>
         </gaia-header>
         <article id="threads-container" class="view-body edit-container" data-type="list">
+          <gaia-fast-list id="threads-list" top="0" bottom="0" caching>
+            <!--<template header>
+              <gaia-sub-header data-time="${daySection}">${section}</gaia-sub-header>
+            </template>-->
+            <template item>
```

Comment:
As I said, it's a trade off, performance against developer ergonomics. You can try using `data-l10n-id` in your template, but it will trigger mutation observer callbacks whilst scrolling, which probably won't be good.

Not sure how else fxos-fastlist can help. Is there a new API that would make this easier for you?